src/components/structures/RoomView.tsx

```diff
@@ -1742,10 +1738,12 @@ export default class RoomView extends React.Component<IProps, IState> {
         // We have successfully loaded this room, and are not previewing.
         // Display the "normal" room view.
 
-        const call = this.getCallForRoom();
-        let inCall = false;
-        if (call && (this.state.callState !== 'ended' && this.state.callState !== 'ringing')) {
-            inCall = true;
+        let activeCall = null;
+        {
+            const call = this.getCallForRoom();
+            if (call && (this.state.callState !== 'ended' && this.state.callState !== 'ringing')) {
+                activeCall = call;
+            }
         }
```

Comment:
no need for the block?

Reply:
I just didn't want the 'call' variable hanging around for the rest of the function

Reply:
ah, would be good to comment on the block then